pages/education/about-gi-bill-benefits/montgomery-active-duty.md

```diff
@@ -13,121 +13,140 @@ aliases:
+## How can I use my MGIB-AD education benefits?
 
-**Train for a specific career, trade, or industry:**
+You can use your GI Bill benefits in many ways to advance your education and training. <br>
+[Find out how to use your GI Bill benefits](/education/about-gi-bill-benefits/how-to-use-benefits/).
 
-- [Vocational/technical training and non-college degree programs](/education/about-gi-bill-benefits/how-to-use-benefits/non-college-degree-programs/)
-- [On-the-job training and apprenticeships](/education/about-gi-bill-benefits/how-to-use-benefits/on-the-job-training-apprenticeships/)
-- [Entrepreneurship training](/education/about-gi-bill-benefits/how-to-use-benefits/entrepreneurship-training/)
-- [Flight training](/education/about-gi-bill-benefits/how-to-use-benefits/flight-training/)
-- [Test Fees](/education/about-gi-bill-benefits/how-to-use-benefits/test-fees/)
+**In some cases, we may help you pay for:**
 
-**Work while you study:**
+  - Remedial courses (classes some students must take to build up their basic skills in math, reading, or English before they can take regular college courses)
+  - Deficiency courses (classes some student must take in order to be admitted to a certain college)
+  - Refresher courses (brief courses that help people review and improve their knowledge in a certain subject area)
 
-- [Co-op training](/education/about-gi-bill-benefits/how-to-use-benefits/co-op-training/)
-- [Work Study](/education/about-gi-bill-benefits/how-to-use-benefits/work-study/)
+[See current payment rates](https://www.benefits.va.gov/gibill/resources/benefits_resources/rate_tables.asp#ch30).
 
-**Take classes from home:**
+## Can I qualify for more than one VA education benefit?
 
-- [Correspondence training](/education/about-gi-bill-benefits/how-to-use-benefits/correspondence-training/)
-- [Independent and distance learning](/education/about-gi-bill-benefits/how-to-use-benefits/online-distance-learning/)
+Yes. You may qualify for more than one education benefit. If you're eligible for more than one, you'll need to decide which to receive. You can't receive benefits under more than one program at a time.
 
-**Get increased or faster payments:**
+We can help you make this decision. Call us at 1-888-GI-BILL-1 (<a href="tel:+18884424551">1-888-442-4551</a>), Monday through Friday, 8:00 a.m. to 7:00 p.m. (<abbr title="eastern time">ET</abbr>). If you have hearing loss, call TTY: 711.
 
-- [Tuition Assistance Top-Up](/education/about-gi-bill-benefits/how-to-use-benefits/tuition-assistance-top-up/)
-- [Accelerated payments for high-tech programs](/education/about-gi-bill-benefits/how-to-use-benefits/high-tech-programs/)
-- [$600 Buy-Up program](/education/about-gi-bill-benefits/montgomery-active-duty/buy-up/): While on active duty, you can pay into this program up to a maximum (the highest amount allowed) of $600 to get increased monthly benefits. If you pay the maximum amount, you can qualify for up to $5,400 in added GI Bill benefits.
+<br>
```

Comment:
The visual display looks like this `<br>` might not be needed? Attaching a screenshot that has what looks like an extra space.

---
![screen shot 2019-02-25 at 9 26 38 am](https://user-images.githubusercontent.com/934879/53351151-4b642a80-38e6-11e9-824b-91c929f11a00.png)

Reply:
This break is intentional because we want the TM info not to appears as if it's a part of the subsection "Can I qualify for more than one VA education benefit?"

Reply:
Sounds good. Thanks for confirming.